src/PrestaShopBundle/Controller/Admin/Improve/International/CurrencyController.php

```diff
@@ -496,6 +496,13 @@ private function getErrorMessages(Exception $e)
                             sprintf('"%s"', $this->trans('Exchange rate', 'Admin.International.Feature')),
                         ]
                     ),
+                CurrencyConstraintException::INVALID_NAME => $this->trans(
+                    'The %s field is not valid',
```

Comment:
It would be better to use the following string: "The %s field is invalid." in the same domain (Admin.Notifications.Error) Thanks!

Reply:
I used the same wording as all other fields in this page If we need to fix this I suggest we do it in a later PR so everything is consistent and we don't delay the stable release